frontend/typescript/tsconfig.dts.json

```diff
@@ -19,7 +19,10 @@
             "../../bazel-out/k8-dbg/bin",
             "../../bazel-out/x64_windows-dbg/bin",
         ],
-        "baseUrl": ".",
+        
+        // https://www.typescriptlang.org/docs/handbook/module-resolution.html#base-ur
```

Comment:
whoops, I missed the `l` on the end of this URL.

Reply:
It's now fixed in the main branch.